tests/post/features/service_configuration.feature

```diff
@@ -0,0 +1,11 @@
+@post @local @ci @persistence
+Feature: Cluster and Services Configurations
+    Scenario: Propagation of Service Configurations to underlying Services
+        Given the Kubernetes API is available
+        And pods with label 'app.kubernetes.io/name=dex' are 'Ready'
+        And we have 2 running pod labeled 'app.kubernetes.io/name=dex' in namespace 'metalk8s-auth' on node 'bootstrap'
+        And we have a 'metalk8s-dex-config' ConfigMap with replicas count '2'
```

Comment:
I think you should use something more generic than that, also here you match directly in the config and `ConfigMap` is not really a keyword here. 

IMO we should have something more generic that you can use for different test something like that
```
we have a 'metalk8s-dex-config' CSC in namespace 'metalk8s-auth' with '.spec.deployment.replicas' equal to '2'
```

So a fixture look like this 
```
we have a '<name>' CSC in namespace '<namespace>' with '<path>' equal to '<value>'
```